repository/index.js

```diff
@@ -0,0 +1,21 @@
+import mu from 'mu';
+
+const mandateeIsCompetentOnFutureAgendaItem = async (endDateOfMandee, mandateeId) => {
+  const query = `
+  PREFIX besluitvorming: <http://data.vlaanderen.be/ns/besluitvorming#>
+
+  ASK WHERE {
+  
+  ?procedurestap besluitvorming:heeftBevoegde <http://kanselarij.vo.data.gift/id/mandatarissen/${mandateeId}> .
```

Comment:
Hier ga je er van uit dat de URI van een mandataris-resource steeds hetzelfde URI-formaat heeft, beëindigd met een id. Dat is niet noodzakelijk zo. Een URI zou er bijvoorbeeld ook als `<http://regering.vlaanderen/jan-janssens>` uit kunnen zien. Elke resource die exposed is door `mu-cl-resources` hoort een `mu:uuid` te hebben. Ook mandatarissen. Het is dus beter als je 
```
?procedurestap besluitvorming:heeftBevoegde ?mandataris .
?mandataris mu:uuid ${sparqlEscapeString(mandateeId)} .
```
gebruikt

Reply:
Done.